### PR TITLE
Add a page noting volunteer teams and their historical members

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Add a list of members of the volunteer teams.

I'm not especially happy with the opening line. I'd like to add more emphasis that it's not just these volunteers who do everything in SR, and that every volunteer helps, but I couldn't find wording I was happy with. Input appreciated!

### CODE REVIEW

**Review by trickeydan — DISMISSED** [review]
I'm requesting changes on the grounds of data protection.

Please confirm that you have permission from individuals to list these names. 

What is the plan for removing the names should the individuals revoke their consent?

**Comment by RealOrangeOne** [comment]
@TomWheal I was originally expecting this to contain a bit about the teams, but that duplicates stuff from the ops manual, which isn't ideal. Yeah moving it into the "Home" section might be better, can give it a look.

@trickeydan not something i'd considered. Revoking consent from git repos is something which no one has worked out, and generally falls under the "not technically feasable" category (most-merge, anyway). From a volunteer perspective, these names are already known, and a historical reference of them could be useful. An alternative, DP-friendly, implementation would be to drop the historical nature, and simply link to the GitHub teams, which enumerate the team and committee members, although I feel that loses something.

File: docs/volunteering/teams.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Volunteer Teams
```
Comment (PeterJCLaw) [inline docs/volunteering/teams.md]: Perhaps use a synonym for teams here if you're concerned about confusion versus competitor teams?
Maybe "organisation", "structure" or "groups"?

File: docs/volunteering/teams.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Student Robotics has 2 teams which organise the operations regarding the wider competition programme. Each team is organised by its respective committee.
+
+- [Competition Team](https://opsmanual.studentrobotics.org/annual-robotics-competition/competition-team)
+- [Kit Team](https://opsmanual.studentrobotics.org/annual-robotics-competition/kit-team)
+
+## 2020
```
Comment (PeterJCLaw) [inline docs/volunteering/teams.md]: Is this the calendar year 2020 or the SR year 2020? It looks like it's meant to be the SR year (which the competition team uses), however we should probably use "SR2020" to clarify that. However of course the Kit Team uses different timings, so using SR years there might be confusing.

Are there other ways of presenting this information which avoids this conflict? (If you've considered some but this still turns out best. please do say!)

File: docs/volunteering/teams.md (hunk 1 above)
Comment (RealOrangeOne) [reply docs/volunteering/teams.md]: The issue is that said groups of people are named "Team", so "Teams" is the logical plural. 

See also https://github.com/srobo/runbook/pull/76#issuecomment-578442154

File: docs/volunteering/teams.md (hunk 2 above)
Comment (RealOrangeOne) [reply docs/volunteering/teams.md]: I'd intended this to be SR years, so yeah I think explicitly noting that is probably a good idea.

AFAIK, the Kit Team tenure falls conveniently around 2 competition cycles, so duplicating the list for the SR year is still correct. Thereby showing who was on the team for a given year, which is still useful information.

**Comment by RealOrangeOne** [comment]
@TomWheal I tried moving the section over to under "Home", although it felt less useful there. The volunteer teams sit quite well under the "Volunteering" section, as we are all volunteers.

I wonder if a separate top-level section, something like "Organisation", which contains the high level organisation things about SR, such as the existence of the ops manual, this teams list, things like that? Although that's probably OOS for this PR. 

Personally it feels fine where it is (for now at least, anyway), although if more people tell me i'm wrong, I'm happy to move it :)

**Comment by TomWheal** [comment]
Having had a bit of a think, I think ive identified why there might be such differing opinions on what the runbook should contain.
My understanding is:
The Ops Manual which describes how the charity operates IE top level teams, budgeting, sponsorship, how to get involved ETC.

The Runbook, details how those top level teams run, for example the way in which communicating with teams has been defined, or the structure of the kit team. 

Its quite possible with the creation of the ops. manual, the whole volunteering section should live on there, along with the content of this PR

**Comment by RealOrangeOne** [comment]
> Its quite possible with the creation of the ops. manual, the whole volunteering section should live on there, along with the content of this PR

This PR shouldn't live on the ops manual, as the ops manual is a living document, rather than an archive. But yes it's definitely possible some bits of content should live there instead. The point of the volunteering section is to be a much less formal version of the ops manual, it obviously shouldn't contradict or duplicate. Ops manual is more about how the charity operates at a high level.

**Comment by PeterJCLaw** [comment]
I'm conflicted on whether this belongs in the Runbook. On the one hand, we don't currently have a better place, yet on the other hand I'm not really sure what useful information this really provides. While in theory you could look at this to know who might know things, it doesn't really have enough detail for that.
I definitely agree this shouldn't be in the ops manual though.

There are also (small) GDPR concerns here about including names (I'm not enough of an expert to know for sure what the impact is, though I do think there's some conflict here -- people do need to be able to easily update their old info and opt-out of things), which might be enough to tip this from being slightly beneficial to not worth it.
I'm not sure we'd need permission from the individuals to list names - this is factual already-public information, being recorded for reasonable purposes. We would need to remove names if asked though. I am not a lawyer though.

**Comment by trickeydan** [comment]
> I'm not sure we'd need permission from the individuals to list names - this is factual already-public information, being recorded for reasonable purposes.

Please could you link *where* this is already public. That is likely also a breach of GDPR.

Reasonable purposes would include holding the data internally, but would not cover publishing the names of past committee members.

**Comment by trickeydan** [comment]
> Also @trickeydan I disagree, your name is already on GitHub being linked to the kit team

Yes, it is. However, GitHub does not currently list the names of previous committees / core team.

**Comment by PeterJCLaw** [comment]
@RealOrangeOne this feels contentious enough (and GDPR skirting enough) that I suggest we drop this. I'm going to close this on those lines; if you feel strongly that you want this back then I suggest starting a discussion in Slack.

**Pull request closed without merging**